packages/calypso-url/src/types.ts

```diff
@@ -0,0 +1 @@
+export type URL = string;
```

Comment:
I believe this typedef should be eventually removed from the package and that we should use `string` instead. In TypeScript with its structural typing, it doesn't make much sense to declare alternative names for types like this.

TypeScript's native typings for `URL` or `URLSearchParams` (in `lib.dom.d.ts`) also use `string` directly.

The name `URL` also conflicts with the `URL` global that's declared both in browsers and Node.

Reply:
Good point. I'll update that.